style.css

```diff
@@ -1,595 +1,1338 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
 
-* {
-  padding: 0;
-  margin: 0;
-  font-family: "Poppins", sans-serif, "Times New Roman", Times, serif;
-  box-sizing: border-box;
-  text-decoration: none;
-  list-style: none;
-}
+@media screen and (max-device-width: 766px) {
+  * {
+    padding: 0;
+    margin: 0;
+    font-family: "Poppins", sans-serif, "Times New Roman", Times, serif;
+    box-sizing: border-box;
+    text-decoration: none;
+    list-style: none;
+  }
```

Comment:
- [ ] Good job, on implementing some of the previous reviewers feedback, but still you are not figure out some issues. Make sure when we use a mobile-first approach, screen size is designed based on mobile-first, so for the media query, we use `min-width` instead of `max-device-width.' If you design for a large-screen, please the media query for the desktop version comes at the end of your CSS, which was used for the mobile version; behind that, the desktop version overrides the mobile version. So kindly use proper syntax for media queries. Generally, we use min and max width. For details [refer to this ](https://www.w3schools.com/css/css3_mediaqueries.asp). Kindly fix the issue :wrench: 

| Figma design | Your design |
|-------------------|-----------------|
| ![Screenshot from 2022-11-03 22-34-51](https://user-images.githubusercontent.com/65129228/199959637-008cce81-35b5-44ff-af07-a7559e4e8ce4.png) | ![Screenshot from 2022-11-04 14-10-47](https://user-images.githubusercontent.com/65129228/199959932-a1c2c932-c8dc-4320-8b73-b3e1e3265608.png) |


Example

- Large mobile screen

```
@media only screen and (min-width: 500px) and (max-width: 767px) {
  .headline-content p {
    max-width: 60%;
  }

```
- Medium size screen

```
@media only screen and (min-width: 768px) and (max-width: 999px) {
  .headline-content h1 {
    font-size: 3.625rem;
  }

```
- Desktop and large screen size

```
@media only screen and (min-width: 1000px) {
   .headline-content h1 {
    font-size: 3.825rem;
  }

```